source/_integrations/squeezebox.markdown

```diff
@@ -46,6 +49,15 @@ transporter_toslink:
         media_content_type: "music"
 ```
 
+## Entities
+
+### Binary sensors
+
+- Needs restart
+  Sever Service would like to be restarted usually to apply updates.
```

Comment:
what do you mean by "usually"? It needs to be restarted? It needs to be restarted on a regular basis?

Reply:
usually its for updates, so comma after restarted?

Reply:
ah!